modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/BasicInfo.tsx

```diff
@@ -36,56 +39,77 @@ export default function BasicInfo({
 	values,
 }: BasicInfoProps) {
 	return (
-		<Card title={Liferay.Language.get('basic-info')}>
-			<InputLocalized
-				error={errors.label}
-				label={Liferay.Language.get('action-label')}
-				name="label"
-				onChange={(label) =>
-					setValues({
-						...values,
-						...(!isApproved &&
-							!values.system && {
-								name: toCamelCase(
-									label[defaultLanguageId] ?? ''
-								),
-							}),
-						label,
-					})
-				}
-				required
-				translations={values.label ?? {[defaultLanguageId]: ''}}
-			/>
+		<>
+			{disableGroovyAction && (
+				<ClayAlert
+					className="lfr-objects__side-panel-content-container"
+					displayType="warning"
+					title="Warning"
+				>
+					{Liferay.Language.get(
+						'groovy-script-validations-are-deactivated-for-your-instance'
+					)}{' '}
+					<a>
+						{Liferay.Language.get(
+							'go-to-system-settings-security-tools-script-management-to-allow-administrators-to-execute-groovy-code'
+						)}
+					</a>
+				</ClayAlert>
+			)}
```

Comment:
Hey @danvb112. What do you think about componentizing this ClayAlert into its own component? I noticed this is used twice.